docker/dev.docker-compose.yml

```diff
@@ -0,0 +1,141 @@
+version: "2"
+services:
+  supervisord:
+    build:
+      context: ..
+      dockerfile: docker/dev.Dockerfile
+    volumes:
+      - ..:/assembl
+    command: "sh -c '. venv/bin/activate && supervisord -n'"
+    depends_on:
+      - database
+      - redis
+      - memcached
+      - elasticsearch
+    ports:
+      - "6543:6543"
+      - "8080:8080"
```

Comment:
Be careful, you are exposing those ports to your network, this is equivalent to `"0.0.0.0:6543:6543"`. Docker add an iptables rule to open those ports on eth0. You probably want `"127.0.0.1:6543:6543"` here.

Reply:
I want access on lan, but I'll restrict to loopback by default